PowerEditor/src/Parameters.cpp

```diff
@@ -6173,10 +6174,21 @@ bool NppParameters::insertTabInfo(const TCHAR *langName, int tabInfo)
 		if (nm && lstrcmp(langName, nm) == 0)
 		{
 			childNode->ToElement()->SetAttribute(TEXT("tabSettings"), tabInfo);
-			_pXmlDoc->SaveFile();
-			return true;
+			return _pXmlDoc->SaveFile();
 		}
 	}
+	auto aExternalLexerConfig = std::find_if(_pXmlExternalLexerDoc.begin(), _pXmlExternalLexerDoc.end(), [langName](TiXmlDocument * doc){
+		auto aLanguaneNode = (doc->FirstChild(TEXT("NotepadPlus")))->FirstChildElement(TEXT("Languages"))->FirstChildElement(TEXT("Language"));
+		auto *element = aLanguaneNode->ToElement();

```

Comment:
The local variable name "aLangua**n**eNode".
I assumed you ment "aLangua**g**eNode" (`g` i/o `n`).